src/tls.c

```diff
@@ -7349,6 +7349,7 @@ int TLSX_KeyShare_Use(WOLFSSL* ssl, word16 group, word16 len, byte* data,
     }
 
     if (data != NULL) {
+        XFREE(keyShareEntry->ke, ssl->heap, DYNAMIC_TYPE_PUBLIC_KEY);
```

Comment:
Check for NULL?

Reply:
In default cases the macro of XFREE in wolfssl/wolfcrypt/types has a built in NULL check. Even so... it does not hurt to have an additional check so that in cases where a user overrides the XFREE macro it will still be checking. I'll add in the change.